content/docs/learning-guide/tutorials/assets/physx-colliders.md

```diff
@@ -0,0 +1,162 @@
+
+## Entity behavior
+
+Before learning about collider types, you should understand the three types of entity behavior that influence the collider asset type you might choose in a given scenario:
+
+* **Static** - Static entities can be collided with, but don't move, and are not affected by PhysX collisions and forces. Static entities can use any collider type.
+* **Kinematic** - Kinematic entities have a **PhysX Rigid Body** component and have movement that is driven by script. Kinematic entities can be collided with, but are not affected by PhysX collisions or forces. Kinematic entities can only use primitive and convex colliders.
+* **Dynamic** - Dynamic entities have a **PhysX Rigid Body** component and simulated movement that results from PhysX collisions and forces. Dynamic entities can only use primitive and convex colliders.
+
+## PhysX collider assets
+
+Collider assets are generated when a source asset is processed and are based on input meshes you specify in Scene Settings. You can use a single input mesh, multiple input meshes, and even specify that multiple input meshes should be merged and welded into a single mesh before the collider asset is generated. The source asset is not altered when the collider asset is generated. The options to combine, clean, weld, and decompose input meshes are performed as part of the process of generating the collider asset, and are discarded when the process job completes.
+
+Because physics simulation can be computationally expensive, it's important to understand the various collider types, their limitations, and the scenarios in which they can be used.
+
+### Triangle Mesh
+
+As the name implies, triangle mesh colliders are composed of triangle meshes. You can create a simplified mesh in a digital content creation (DCC) application specifically for the collider, or generate a triangle mesh collider based on the render mesh of the asset. Triangle mesh colliders can closely approximate a complex render mesh, but can only be used with static entities. Because triangle mesh colliders can contain many vertices, and the collider asset is not required to be convex, they generally incur a higher performance cost than primitive colliders.
+
+Triangle mesh colliders can have multiple physics material assignments. Because the collider is derived from the input mesh, one physics material can be applied to the collider per each material assignment. This is particularly useful for ground and floor colliders that transition between materials such as concrete and dirt, or wood planks and carpet.
+
+In the image below, a triangle mesh collider asset is offset to the right of the input mesh asset that generated the collider. Note that the collider asset (right) accurately represents the render mesh (left) and that the collider supports separate physics materials for the front and back faces (magenta wireframe) and side faces (teal wireframe) of the mesh.

```

Comment:
Changed all these examples to something like:

Colliders occupy the same physical space as the input mesh when they are generated. In the image below, a triangle mesh collider asset is offset to the right of the input mesh asset that generated the collider so that you can clearly view the collider.